_projects/workflow-endpoint-registry.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: page_project
+title: A common workflow registry of compute endpoints and applications
+date: 2020-07-02
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: starting
+topics:
+  - prog_lang
+keywords:
+  -
+head: katz_d
+members:
+  - badia_r
+  - chard_k
+  - ejarque_j
+---
+{% comment %}
+================================
```

Comment:
Not familiar with the system which generates the website from the github repository. It is appearing in the github visualization but I suppose this part will be ignored when generating the web, right?

Reply:
This will be the meta-data of the project description. See https://jlesc.github.io/projects/interop_chameleon_g5k/ on the right for an example.